tutorials/beginner/installing.md

```diff
@@ -1,27 +1,55 @@
 ---
 layout: tutorial-beginner
+title: Installing Extension
 ---
 
 {% include base.html %}
 
-# To be done
+# Installing an Extension
```

Comment:
Not sure about that. The menu in PaperUI has it as "Extension" and so does the ESH documentation. Perhabs the documentation should follow the implementation and always use "Extension"?